libr/anal/session.c

```diff
@@ -0,0 +1,68 @@
+/* radare - LGPL - Copyright 2017 - rkx1209 */
+#include <r_anal.h>
+#include <r_types.h>
+#include <r_util.h>
+#include <r_bind.h>
+
+R_API void r_anal_esil_session_free(RAnalEsilSession *session) {
+	free (session->data);
+	free (session);
+}
+
+R_API void r_anal_esil_session_list(RAnalEsil *esil) {
+	if (!esil || !esil->sessions) {
+		return;
+	}
+	eprintf ("%d\n", r_list_length (esil->sessions));
```

Comment:
you may use esil->anal->cb_printf ("..."); this is hooked to r_cons_printf via rcore